MedicalApp.BuisnessServices/Mappers/Implementation/PatientMapper.cs

```diff
@@ -30,7 +41,15 @@ public void MapFromEntity(PatientEntity item, Patient targetItem)
                 targetItem.FirstName = item.FirstName;
                 targetItem.LastName = item.LastName;
                 targetItem.Age = item.Age;
-                targetItem.RoomId = item.RoomId;
+                //targetItem.RoomId = item.RoomId;
```

Comment:
Don't leave comment's if you don't need them. Exception: //TODO statement

Reply:
The comment was removed